server/mergin/sync/public_api_controller.py

```diff
@@ -193,6 +193,7 @@ def add_project(namespace):  # noqa: E501
 
         p = Project(**request.json, creator=current_user, workspace=workspace)
         p.updated = datetime.utcnow()
+        db.session.flush()
```

Comment:
Is not enough just to call session.add here which would autoflush changes?

Also is this the only place which causes issues for project version created? How about clone/push/push finish?

Reply:
Some tests are failing without flushing the DB.
However, it's probably a good practice to add the instance to the DB before flushing the DB.

The clone controller doesn't fail :man_shrugging: but let's add the project to DB right after initialization.

We do not create a new project when pushing changes.